src/cascadia/TerminalApp/ColorScheme.idl

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "../TerminalSettings.idl";

```

Comment:
you don't need this. you're using `IControlSettings` below, not `TerminalSettings`.